docs/source/gbd2019_models/risk_exposures/bmi/index.rst

```diff
@@ -0,0 +1,112 @@
+.. _2019_risk_bmi:
+
+======================================
+Body Mass Index
+======================================
+
+
+Risk Exposure Overview
+----------------------
+
+Body mass index (BMI) is a person’s weight in kilograms divided by the square of height in meters (weight (kg) / [height (m)]\ :sup:`2`\). BMI is an inexpensive and easy screening method; values are frequently divided into categories: underweight (<18.5 kg/m\ :sup:`2`\), healthy weight (18.5 to 24.9 kg/m\ :sup:`2`\), overweight (25.0 to 29.9 kg/m\ :sup:`2`\), and obesity (>=30.0 kg/m\ :sup:`2`\). BMI does not measure body fat directly, but it is moderately correlated with more direct measures of body fat. BMI has also been shown to be correlated with various metabolic and disease outcomes. BMI can be a screening tool, but it does not diagnose the body fatness or health of an individual.
```

Comment:
"obese" instead of "obesity" as a category name sounds better to me.  if "obesity" is the term of art, who am I to say we should change it, though?

Reply:
Good point! I'll change "obesity" to "obese", since the CDC reference we are citing also uses "obese" instead of "obesity".